.github/pull-request-template.md

```diff
@@ -4,4 +4,18 @@ Request Prow to automatically lint any go code in this PR:
 /lint
 -->
 
+**What this PR does, why we need it**:
+
+<!--
+*Automatically closes linked issue when PR is merged.
+Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
+-->
+**Which issue(s) this PR fixes**:
 Fixes #
+
+**Special notice to reviewersr**:
+
+**Has this pr changed for users**:
+```
+
+```
```

Comment:
```suggestion
```

Reply:
I didn't delete this because it's markdown syntax, like this:
```
xxxxxxxxxxxxxxxxxxxxxxxxxxx
```